picasso/src/test/java/com/squareup/picasso3/SourceBufferingInputStreamTest.java

```diff
@@ -45,6 +47,61 @@
     assertEquals(13, stream.available());
   }
 
+  @Test public void read() throws IOException {
```

Comment:
I've updated tests so that `read` and `mark`/`reset` methods should comply with the `InputStream` requirements.